website/source/docs/job-specification/migrate.html.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: "docs"
+page_title: "migrate Stanza - Job Specification"
+sidebar_current: "docs-job-specification-migrate"
+description: |-
+  The "migrate" stanza specifies the group's migrate strategy. The migrate
+  strategy is used to control the job's behavior when it is being migrated off
+  of a draining node.
+---
+
+# `migrate` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> **migrate**</code>
+    </td>
+    <td>
+      <code>job -> group -> **migrate**</code>
+    </td>
+  </tr>
+</table>
+
+The `migrate` stanza specifies the group's strategy for migrating off of
+[draining][drain] nodes. If omitted, a default migration strategy is applied.
+If specified at the job level, the configuration will apply to all groups
+within the job. Only service jobs with a count greater than 1 support migrate
+stanzas.
+
+```hcl
+job "docs" {
+  migrate {
+    max_parallel     = 1
+    health_check     = "checks"
+    min_healthy_time = "10s"
+    healthy_deadline = "5m"
+  }
+}
+```
+
+When one or more nodes are draining, only `migrate.max_parallel` allocations
```

Comment:
do we need the migrate. prefix? It is the doc page for this stanza

Reply:
yeah, it looks gross and seems useless. removing.